tests/tracer/test_sampling_decision.py

```diff
@@ -0,0 +1,52 @@
+import pytest
+
+from ddtrace.context import Context
+from ddtrace.internal.sampling import SamplingMechanism
+from ddtrace.internal.sampling import _hash_service
+from ddtrace.internal.sampling import update_sampling_decision
+from tests.utils import override_global_config
+
+
+@pytest.fixture()
+def context():
+    yield Context()
+
+
+@pytest.mark.parametrize(
+    "service,expected",
+    [
+        ("service", "9df6b026a8"),
+        (u"\x80service", "069dd5df76"),
+        (b"\x80service", None),
```

Comment:
why? bytes?

Reply:
Should we handle bytes differently? This check was to clarify the hashing assumes `service` is a string literal and not a bytes literal.